### ISSUE TITLE
Good a11y work

### ISSUE THREAD

**Issue by oliverjam** [fac25/week2-MarkupProject-Alex-Joe#41]
It looks like you took accessibility seriously; nice not to see anything missing or broken for users with extra needs ✨
